src/schema/firefox-schemas.js

```diff
@@ -0,0 +1,189 @@
+const FLAG_PATTERN_REGEX = /^\(\?[im]*\)(.*)/;
+
+// Reference some functions on inner so they can be stubbed in tests.
+export const inner = {};
```

Comment:
This is definitely a hack. I don't think you can stub the functions with sinon if they aren't attached to an object though, unfortunately.

Since these functions are internal to this module they need to be attached to something to get stubbed.

I think having them on the `inner` object shows that they're private, but I'm not sure it matters much. If it weren't for the sinon issue I would have exported them regularly.